### ISSUE TITLE
Unbreak eyeglass asset helper

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#2197]
#2171 introduced a regression that is causing many eyeglass tests to fail for the asset helper. 

```sh
assets
    ✓ should give an error when an asset is not found (82ms)
    1) should let an app refer to it's own assets
    ✓ asset importer should delegate to custom importer
    2) should import a module's assets
    3) should import own assets from within a module
    4) should allow httpPrefix for app assets
    5) should allow a global httpPrefix for all assets
    6) should allow a relative URLs
    7) should allow a relative URLs when httpRoot is set
    8) should nest a asset path entry http prefix inside the global httpPrefix
    ✓ register-asset() works
    9) allows url mangling
    10) allows installing assets
    11) allows installing assets
    12) handles asset installer errors
    13) should not install into the httpRoot
    ✓ should manually install assets
    ✓ should handle an installer error
    14) should handle undefined file being passed to install
    15) should handle an error in a resolver
    16) should handle a sass error in a resolver
    ✓ should give an error when a module does not have assets
    ✓ should give an error when a module does not exist
    ✓ can pretty print an asset path entry
    ✓ can assign custom glob opts to an asset path entry
    ✓ asset path entries must be directories
    17) should allow uri fragments
    18) asset-uri should return the uri only, not wrapped in url()
    19) support assets specified via options
    20) should normalize URI directory traversal
    21) should support odd character in file names

  136 passing (2s)
  1 pending
  21 failing
```

@chriseppstein I don't know much about the internals of eyeglass. I would appreciate some help from yourself or @eoneill to get a reduced test case together.

I believe this is the last blocker for a 3.4RC.

**Comment by xzyfer** [comment]
Patch provided upstream - https://github.com/sass-eyeglass/eyeglass/pull/145

**Comment by xzyfer** [comment]
Fixed in eyeglass@1.1.2
